lib/puppet.rb

```diff
@@ -18,6 +18,8 @@
 require 'puppet/external/pson/common'
 require 'puppet/external/pson/version'
 require 'puppet/external/pson/pure'
+require 'gettext-setup'
+require 'locale'
```

Comment:
I think when running from gems we get `locale` installed as a transitive dependency  - `gettext-setup` => `gettext` => `locale`. Since we require this here, thoughts on expressing a dependency on `locale` in `ext/project_data.yaml` too?